roles/delay-prediction-service/templates/delay-prediction-service.service

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=pull bus positions from Thingsboard, predict delays & positions, send GTFS-RT to MQTT
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Restart=always
+RestartSec=5
+
+Environment=THINGSBOARD_HOST="{{ thingsboard_api_url }}"
```

Comment:
I can decrypt the secrets. Can you check if `thingsboard_api_url` is just the host (e.g. `thingsboard.cloud`) or a full URL (e.g. `https://thingsboard.cloud/api/foo`)?

Reply:
It's a full URL:

```
Environment=THINGSBOARD_HOST="https://portal.mhascaro.com/api"
```